primitives/src/transaction.rs

```diff
@@ -76,8 +76,12 @@ impl fmt::Display for Version {
 #[cfg(feature = "arbitrary")]
 impl<'a> Arbitrary<'a> for Version {
     fn arbitrary(u: &mut Unstructured<'a>) -> arbitrary::Result<Self> {
```

Comment:
Can you throw a comment on this function please, something like `// Prefer the normal version numbers`. Same for the `Sequence` one, the idea being to save the reader from wondering _why_ we are doing this and not just returning an arbitrary value.

Reply:
Sure, i'll update this in a bit once i get home